CoffeeMachineSimulator/CoffeeMachineSimulator.Tests/Services/CoffeeServiceTests.cs

```diff
@@ -11,6 +12,9 @@ namespace CoffeeMachineSimulator.Tests.Services
     public class CoffeeServiceTests
     {
         private CoffeeService coffeeService;
+        private IEnumerable<CoffeeModel> coffeeModels;
+        private object coffeeId;
+        private object coffee;
```

Comment:
do you need these objects ? if not, please delete them. All the time when coding, follow YAGNI (you ain't gonna need it) principle. This says that if you don't need an object, do not create it